Resources/Prototypes/_RMC14/Entities/rmc_planets.yml

```
@@ -61,6 +61,28 @@
       Telecommunications Array: Repairs required
 
       No further information available at this time."
+    survivorJobs:
+    - RMCSurvivorCommandingOfficer: 1
+    - CMSurvivor: 7 #Should remain equal to RMCSurvivorMaximum. Setting to -1 makes this spawn normal civ survs for some reason.
```

```suggestion
    - CMSurvivor: -1
```
Change this back to -1, i know the issue and im PRing a fix